jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/MetaInfConfiguration.java

```diff
@@ -166,8 +168,8 @@ public void findAndFilterContainerPaths(final WebAppContext context) throws Exce
         if (classPath != null)
         {
             Stream.of(classPath.split(File.pathSeparator))
-                .map(URIUtil::toURI)
-                .filter(uriPatternPredicate)
+                .map(resourceFactory::newResource)
```

Comment:
We should modify the method's javadoc to specify it mounts jars.

Reply:
I've cleaned up the ee10 version and added javadoc. I'd prefer minimal changes to ee9